.github/workflows/securityAlertsReview.yml

```diff
@@ -0,0 +1,194 @@
+name: Security Alerts Review
+
+on:
+  push:
+    paths:
+      - 'src/**.sol'
+      - ".github/workflows/securityAlertsReview.yml"
```

Comment:
why is the path of the git action itself here?

Reply:
removed